export const allCasesLogout = () => {
  try {
    return 'Todo bien en el logout';
  } catch (error) {
    throw Error('Todo mal en el logout');
  }
};
